core/services/chainlink/application.go

```diff
@@ -196,6 +198,18 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	restrictedHTTPClient := opts.RestrictedHTTPClient
 	unrestrictedHTTPClient := opts.UnrestrictedHTTPClient
 
+	var prflr *pyroscope.Profiler
+	if cfg.PyroscopeServerAddress() != "" {
+		globalLogger.Info("Pyroscope (automatic pprof profiling) is enabled")
+		var err error
+		prflr, err = logger.StartPyroscope(cfg)
+		if err != nil {
+			return nil, errors.Wrap(err, "starting pyroscope (automatic pprof profiling) failed")
+		}
+	} else {
+		globalLogger.Info("Pyroscope (automatic pprof profiling) is disabled")
```

Comment:
Would we miss this if it were reduced to debug level?

Reply:
I would need to test it and see if we reduce the log level to debug.

Reply:
We would still capture this in our disk logs, since everything >= DEBUG is capture there.

The only way for us to miss this in the console logs, is if we set the log level to be above INFO, but the disk logs will retain this information @jmank88.

Reply:
Yeah I mean that this seems like it is just noise for everyone who doesn't care about this feature, so let's change it to debug level.